### PR TITLE
Add call for Noah's greeting

### PR DESCRIPTION
**Pull request by nwconstable**
Adds the call for Noah's greeting in the builder

### CODE REVIEW

**Review by JacobJenness — DISMISSED** [review]
LGTM

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public String generateOutput() {
 
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
+    builder.append(noahSaysHello());
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java]: This is properly alphabetized.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -33,4 +34,8 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+  private String noahSaysHello() {
+    return "Noah says 'Hey!'\n";
+  }
+
```
Comment (day00090) [inline src/main/java/hellos/Hellos.java:94]: The formatting looks correct, and the tests pass. I think you're ready to go!

**Review by day00090 — DISMISSED** [review]

**Review by RichardLussier — DISMISSED** [review]
Looks good to go

**Review by IsabelleHjelden — DISMISSED** [review]

File: src/main/java/hellos/Hellos.java [hunk 3]
```diff
@@ -26,6 +26,7 @@ public String generateOutput() {
     builder.append(joshSaysHello());
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
+    builder.append(noahSaysHello());
```
Comment (lam00071) [inline src/main/java/hellos/Hellos.java:37]: Hi there hehe doing good job bro.

Comment (lam00071) [reply src/main/java/hellos/Hellos.java:37]: Right alphabetized too.

**Review by lam00071 — DISMISSED** [review]
Let's gooooo

File: src/main/java/hellos/Hellos.java [hunk 4]
```diff
@@ -43,12 +44,18 @@ private String kkSaysHello() {
...
+    return "Noah says 'Hey!'\n";
+}
+  
+ private String jeffSaysHello(){
     return "Jeff says 'Hello!'\n";
   }
 
   private String cassieSaysHello(){
     return "Cassie says 'Hello!'\n";
+
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: I wouldn't add a blank line here.

**Review by NicMcPhee — CHANGES_REQUESTED** [review]

File: src/main/java/hellos/Hellos.java [hunk 5]
```diff
@@ -49,11 +50,16 @@ private String thomasSaysHello() {
...
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]:
```suggestion
```

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
You still have that blank line in Cassie's function that probably should be removed.

**Review by NicMcPhee — APPROVED** [review]
Looks great. In the "real work" I would probably do the change to the MD file at least as a separate commit, and possibly a separate PR, since it's a totally unrelated change. For this, though, I'm totally fine with approving it (and appreciate you noticing and fixing the typo).

**Review by day00090 — DISMISSED** [review]
I don't see any issues with the formatting or positioning of the changes.

**Review by RichardLussier — DISMISSED** [review]
Looks good

**Review by NicMcPhee — DISMISSED** [review]

**Review by IsabelleHjelden — DISMISSED** [review]
Looks good

**Review by NicMcPhee — DISMISSED** [review]

**Review by biruk741 — DISMISSED** [review]

**Review by IsabelleHjelden — APPROVED** [review]
Good to go

**Pull request merged**